### PR TITLE
Rename IbisSource to FrankFramework

### PR DESCRIPTION
**Pull request by nielsm5**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,28 +43,22 @@
...
 		</developer>
 		<developer>
 			<name>Sergi Philipsen</name>
-			<email>sergi@wearefrank.nl</email>
-			<organization>WeAreFrank!</organization>
-			<organizationUrl>https://wearefrank.nl</organizationUrl>
+			<email>sergi@frankframework.nl</email>
+			<organization>Frank!Framework</organization>
+			<organizationUrl>https://frankframework.org</organizationUrl>
 		</developer>
```
Comment (jkosternl) [inline pom.xml:55]: We should add Martijn too, since he build most of it, right?

File: pom.xml [hunk 2]
```diff
@@ -109,6 +103,46 @@
 						</execution>
 					</executions>
 				</plugin>
+				
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-javadoc-plugin</artifactId>
+					<executions>
```
Comment (jkosternl) [inline pom.xml:110]: You've deleted `<version>3.6.3</version>` in the first pom, but didn't add it here, so it falls back to an older not pinned version. 
Please add this line here: `<version>3.6.3</version>`

**Review by jkosternl — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (jkosternl) [reply pom.xml:55]: Oh oops... the collapsing did work too good 😉 .. sorry.

**Review by jkosternl — APPROVED** [review]
Good changes!

File: pom.xml [hunk 3]
```diff
@@ -43,28 +43,22 @@
...
+			<email>niels@frankframework.org</email>
+			<organization>Frank!Framework</organization>
+			<organizationUrl>https://frankframework.org</organizationUrl>
 		</developer>
 		<developer>
 			<name>Sergi Philipsen</name>
-			<email>sergi@wearefrank.nl</email>
-			<organization>WeAreFrank!</organization>
-			<organizationUrl>https://wearefrank.nl</organizationUrl>
+			<email>sergi@frankframework.nl</email>
```
Comment (philipsens) [inline pom.xml:52]:
```suggestion
			<email>sergi@frankframework.org</email>
```

**Review by philipsens — CHANGES_REQUESTED** [review]

**Review by philipsens — APPROVED** [review]

**Pull request merged**